lib/date_time/date_time.c

```diff
@@ -193,10 +196,41 @@ static int sntp_time_request(struct ntp_servers *server, uint32_t timeout,
 	return err;
 }
 
+#if defined(CONFIG_LTE_LINK_CONTROL)
+static bool is_connected_to_lte(void)
+{
+	int err;
+	enum lte_lc_nw_reg_status reg_status;
+
+	err = lte_lc_nw_reg_status_get(&reg_status);
+	if (err) {
+		LOG_WRN("Failed getting LTE network registration status, error: %d", err);
+		return false;
+	}
+
+	if (reg_status == LTE_LC_NW_REG_REGISTERED_EMERGENCY ||
+	    reg_status == LTE_LC_NW_REG_REGISTERED_HOME ||
+	    reg_status == LTE_LC_NW_REG_REGISTERED_ROAMING) {
+		return true;
+	}
+
+	return false;
+}
+#endif /* defined(CONFIG_LTE_LINK_CONTROL) */
+
 static int time_NTP_server_get(void)
 {
 	int err;
 
+#if defined(CONFIG_LTE_LINK_CONTROL)
+	if (!is_connected_to_lte()) {
+		LOG_WRN("Not connected to LTE, skipping NTP UTC time update");
```

Comment:
Can this be changed to LOG_DBG so as to silently skip this round of check?

Reply:
fixed